docs/content/configuration/authentication.md

```diff
@@ -0,0 +1,45 @@
+## Authentication
```

Comment:
You need front matter here to register it as a page.

Check out another non index page for an example.